scripts/jq/1.5-239278fd/script.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+MASON_NAME=jq
+MASON_VERSION=239278fd3a02dc1ae0521a7a57d573dbf977b2d9
+MASON_LIB_FILE=bin/jq
+
+. ${MASON_DIR}/mason.sh
+
+function mason_load_source {
+    export MASON_BUILD_PATH=${MASON_ROOT}/.build/${MASON_NAME}-${MASON_VERSION}
+
+    git clone git://github.com/stedolan/${MASON_NAME}.git ${MASON_BUILD_PATH}
+    cd ${MASON_BUILD_PATH}
+    git checkout ${MASON_VERSION}
+
+    cd ../
```

Comment:
Direct git clone works - nice job figuring this out - however I'd prefer to keep the `mason_download` usage when possible, for consistency. In this case I think you can follow the lead of https://github.com/mapbox/mason/blob/master/scripts/protozero/ccf6c39/script.sh which is an example of pulling from a specific gitsha while still using `mason_download`.

ooooo, now I'm seeing the `submodule update --init`. That must be why you are forced to git clone instead of pulling from a tag? If that is the case then ignore my above comment. Though I will say that here is another example of pulling using git: https://github.com/mapbox/mason/blob/master/scripts/mapnik/f02a25901/script.sh#L10-L14. The only real substantial difference is pulling via `https` rather than `git`, but wanted to provide that example nonetheless

Reply:
👍 Yeah I had used mason download tied to the commit originally but then discovered I needed the submodules